docs/website/contents/for-developers/OperationalCertificateIssueNumber.md

```diff
@@ -0,0 +1,81 @@
+## Introduction 
+
+This document explains and motivates operational certificate issue numbers (aka opcert issue numbers), their relationship with header validity, block tiebreakers, and short forks.
+
+## Syntax
+
+- {FooBar} is the definition of FooBar, useful for navigation via CTRL+F.
+- X, Y, and Z are variable names whose scope does not extend beyond this document.
+
+## Definition
+
+In the most granular sense, a party producing Cardano blocks is identified by a pair X and Y.
+
+- {X} is a {ColdKey}; the ledger state includes the public half via the stake pool's registration.
+- {Y} is an {OperationalCertificateIssueNumber}; the Cardano header includes one, signed by X.
+  (The initialism {OCIN} abbreviates OperationalCertificateIssueNumber in this document but is not broadly used outside of it.)
+
+An {OperationalCertificate} (aka opcert) grants a {HotKey} the right to issue blocks on behalf of some X.
+An OperationalCertificate is present in each Cardano header and contains the OCIN Y.
+The ColdKey-HotKey indirection enables the stake pool operator (aka {SPO}) to store their ColdKey offline, so that it's plausible ColdKeys will never be acquired by the adversary excepty through physical access or coercion (eg burglary/bribery).
+
+## Consequences and Motivation
+
+The benefit of identities including an OCIN Y rather than merely being determined by a ColdKey X is that an SPO can increment their OCIN whenever they suspect the adversary could have acquired their current HotKey (eg they discover evidence of unauthorized software running on their block-producing node).
+This response will be effective because the header validity and tiebreaker rules involve the OCIN.
+
+- The protocol state maintains a mapping from X to Z, the OCIN of the youngest header issued by X on this chain.
+  The protocol rules allow exactly two identities to issue a header extending some chain when X is elected, either Y=Z or Y=Z+1.
+- The Cardano tiebreaker rules favor the greater OCIN when comparing two headers from X in the same slot.
+  Without this preference in the tiebreaker, whether X is able to increment their OCIN on the honest chain is merely a network race between the adversary's header/block and the SPO's header/block.
```

Comment:
It might be worth to mention (maybe in a footnote) that the VRF tiebreaker can't help here because it is determined by {ColdKey} and the slot, which both are identical here.